src/content/docs/wallets.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Web Monetization-compatible wallets
+lastUpdated: 2025-02-18
+---
+
+import { Steps, Tabs, TabItem } from '@astrojs/starlight/components'
+import { LinkOut } from '@interledger/docs-design-system'
+
+The following providers offer Web Monetization-compatible digital wallets. You must have a wallet address or payment pointer from a compatible wallet to send and receive payments.
+
+Be sure to select a wallet that's available in your **region** and supports your preferred **currency**. See the [Limitations](#limitations) section for additional details.
+
+## Wallet providers
+
+| Provider | Payment address label | Available in | Supported fiat currencies |
```

Comment:
Maybe we should remove the "Payment address label" column, as we're explaining the specifics in "Where is my payment address?" section below already?

Like for a user coming to this table, are they ready to know about wallet address/payment pointer at this stage?